### ISSUE TITLE
NTR: asymmetrical gluteal crease

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4178]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Asymmetrical gluteal crease

### 2. Synonyms
Asymmetrical buttock crease
Asymmetrical gluteal sulcus
Asymmetrical horizontal gluteal crease

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
The presence of an asymmetrical gluteal crease, the horizontal crease formed by the inferior aspect of the buttocks and the posterior upper leg.
PMID:29701087 

'has part' some 
    (asymmetrical
     and ('inheres in' some 'gluteal sulcus')
     and ('has modifier' some abnormal))

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
HP_0005107

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
Hip dysplaisa: https://hipdysplasia.org/for-physicians/pediatricians-and-primary-care-providers/examination-after-age-3-months/

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
https://orcid.org/0000-0001-5208-3432

**Comment by nicolevasilevsky** [comment]
@drseb could you please MIREOT UBERON_0036014 gluteal sulcus
@drseb and @pnrobinson do you think this is the best parent? HP_0005107 'Abnormal sacrum morphology'?

**Comment by pnrobinson** [comment]
The gluteal sulcus is basically the crease under one's butt...and so if it is asymmertric it can be because of asymmetric bones or asymmetric muscles. I am not sure of the best place to put the parentage. @cmungall @mellybelly @drseb any ideas?

**Comment by pnrobinson** [comment]
THis was HP_0410260
I am moving this to abn of the musculature of the lower limbs -- it is not perfect here, but sacrum is completely wrong.
